src/api/QdmCalculationService.ts

```diff
@@ -1,14 +1,46 @@
 import { Calculator } from "cqm-execution";
-import { CqmMeasure } from "cqm-models";
+import { CqmMeasure, IndividualResult } from "cqm-models";
+import {
+  Group,
+  Measure,
+  PopulationExpectedValue,
+  PopulationType,
+  TestCase,
+} from "@madie/madie-models";
+import * as _ from "lodash";
+import { ExecutionStatusType } from "./CalculationService";
+import { GroupPopulation } from "@madie/madie-models/dist/TestCase";
+import { isTestCasePopulationObservation } from "../util/Utils";
+import { getPopulationTypesForScoring } from "../util/PopulationsMap";
 
-export class QdmCalculationService {
-  async calculateQdmTestCases(cqmMeasure: CqmMeasure, patientJson) {
-    let patients = [];
-    patients.push(patientJson);
+export enum CqmPopulationType {
+  IPP = "initialPopulation",
+  DENOM = "denominator",
+  DENEX = "denominator-exclusion",
+  DENEXCEP = "denominator-exception",
+  NUMER = "numerator",
+  NUMEX = "numerator-exclusion",
+  MSRPOPL = "measure-population",
+  MSRPOPLEX = "measure-population-exclusion",
+  OBSERV = "measure-observation",
```

Comment:
I see that all of them are not following the camel case syntax is this expected ?

Reply:
so, I copied this over from the QDM execution written in EditTestCase for single test case execution. I should have checked that it was written properly, as it was NOT! So, I'll be updating these.
It works for IP, Denom and Numer, but anything hyphenated it fails.
I'm also adding an extra unit test with at least one hyphenated population.

Reply:
this has been corrected and an extra unit test added

Reply:
thanks for the clarification Nate